frontend/ui/components/Topbar/_general/Selector/Selector.jsx

```diff
@@ -0,0 +1,20 @@
+import Select from 'react-select';
+import { selectStyles } from './Selector.style.js';
+import { DropdownIndicator } from './DropdownIndicator/DropdownIndicator.jsx';
+import { SingleValue } from './SingleValue/SingleValue.jsx';
+
+
+export const Selector = ({ onChange, options, defaultValue }) => {
+  const style = selectStyles();
+
+  return (
```

Comment:
Add styles for No Options
<img width="244" alt="image" src="https://github.com/user-attachments/assets/fe89f2df-a988-47af-aa5b-2fd2dcffe2c4">